docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG CUDA_VERSION=cu12
 
-FROM nvidia/cuda:12.4.1-devel-ubuntu22.04 AS cu12
+FROM nvidia/cuda:12.0.0-devel-ubuntu22.04 AS cu12
```

Comment:
@ColorfulDick hi, can we revert this line and keep using cuda12.4.1

Reply:
> @ColorfulDick hi, can we revert this line and keep using cuda12.4.1

i have change in newst commit;